package.json

```diff
@@ -42,79 +42,80 @@
     "label tool"
   ],
   "devDependencies": {
-    "@babel/core": "^7.17.8",
-    "@babel/plugin-proposal-class-properties": "^7.10.4",
-    "@babel/preset-env": "^7.16.11",
-    "@babel/preset-react": "^7.16.7",
+    "@babel/cli": "^7.23.0",
+    "@babel/core": "^7.23.3",
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
+    "@babel/preset-env": "^7.23.3",
+    "@babel/preset-react": "^7.23.3",
     "@svgr/webpack": "^6.2.1",
     "@testing-library/jest-dom": "^5.16.2",
+    "@testing-library/react": "^12.1.2",
     "@testing-library/react-hooks": "5.x",
     "@wojtekmaj/enzyme-adapter-react-17": "^0.6.0",
     "antd-dayjs-webpack-plugin": "^1.0.6",
     "babel-eslint": "^10.1.0",
     "babel-jest": "^27.5.1",
-    "babel-loader": "^8.2.3",
-    "babel-plugin-styled-components": "^2.0.6",
+    "babel-loader": "^8.3.0",
+    "babel-plugin-module-resolver": "^5.0.0",
+    "babel-plugin-styled-components": "^2.1.4",
     "clean-webpack-plugin": "^4.0.0",
     "css-loader": "^5.2.4",
     "css-minimizer-webpack-plugin": "^3.4.1",
     "enzyme": "^3.11.0",
     "enzyme-to-json": "^3.5.0",
     "eslint": "^7.9.0",
     "eslint-config-standard": "^16.0.3",
-    "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-import": "^2.29.0",
+    "eslint-plugin-indent-class-properties": "^1.0.3",
     "eslint-plugin-jest": "^24.7.0",
-    "eslint-plugin-jsx-a11y": "^6.3.1",
+    "eslint-plugin-jsx-a11y": "^6.8.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-notice": "^0.9.10",
     "eslint-plugin-only-error": "^1.0.2",
     "eslint-plugin-promise": "^5.2.0",
-    "eslint-plugin-react": "^7.29.4",
-    "eslint-plugin-react-hooks": "^4.3.0",
+    "eslint-plugin-react": "^7.33.2",
+    "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-standard": "^4.0.1",
     "esm": "^3.2.25",
     "flush-promises": "^1.0.2",
     "fs-extra": "^10.1.0",
     "gh-pages": "^5.0.0",
-    "html-webpack-plugin": "^5.5.0",
+    "html-webpack-plugin": "^5.5.3",
     "husky": "^4.3.8",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^27.5.1",
-    "jest-canvas-mock": "^2.3.1",
-    "less": "^4.1.2",
+    "jest-canvas-mock": "^2.5.2",
+    "less": "^4.2.0",
     "less-loader": "^10.2.0",
-    "mini-css-extract-plugin": "^2.6.0",
+    "mini-css-extract-plugin": "^2.7.6",
     "postcss": "^8.4.12",
     "postcss-flexbugs-fixes": "^5.0.2",
     "postcss-loader": "^6.2.1",
     "postcss-preset-env": "^7.4.3",
+    "postcss-scss": "^4.0.9",
+    "postcss-styled-syntax": "^0.5.0",
     "postcss-syntax": "^0.36.2",
-    "prettier": "^2.1.1",
```

Comment:
why to drop it?

Reply:
If you are using `prettier` locally then it should be installed locally on your machine otherwise `prettier` should be a part of SDLC. Right now it's not.